player/command.c

```diff
@@ -2278,12 +2278,12 @@ static int property_imgparams(struct mp_image_params p, int action, void *arg)
     for (int i = 0; i < desc.num_planes; i++)
         bpp += desc.bpp[i] >> (desc.xs[i] + desc.ys[i]);
 
-    // Alpha type is not supported by FFmpeg, so MP_ALPHA_AUTO may mean alpha
-    // is of an unknown type, or simply not present. Normalize to AUTO=no alpha.
-    if (!!(desc.flags & MP_IMGFLAG_ALPHA) != (p.alpha != MP_ALPHA_AUTO)) {
-        p.alpha =
-            (desc.flags & MP_IMGFLAG_ALPHA) ? MP_ALPHA_STRAIGHT : MP_ALPHA_AUTO;
-    }
```

Comment:
Which one exactly? 

```
int fmt = p.hw_subfmt ? p.hw_subfmt : p.imgfmt;
if (fmt) {
```

If we don't have `fmt` it will not be infered. `vo_gpu_next` doesn't set `fmt`, because there is no good way to map it, instead it sets `imgfmt_name` and `alpha` directly. It does not set `average-bpp`, I have a patch for that, but it is not that of a useful value anyway...